snakemake/executors/__init__.py

```diff
@@ -466,6 +490,9 @@ def get_python_executable(self):
     def get_envvar_declarations(self):
         return ""
 
+    def get_resources_declarations(self, job):
+        return ""
```

Comment:
Let's add a comment here that explains why the CPU executor should not pass resource declarations to spawned run-directive jobs.

Reply:
Are you sure that it should not? I currently don't see why. I think it actually should.

Reply:
Can't remember why I did that, but I'll remove it and see if it passes the tests